client-rpc/src/handler.rs

```diff
@@ -63,11 +65,27 @@ impl RpcHandler {
             fee_policy.clone(),
             tendermint_client.clone(),
         )?;
+        // wait for first block to appear
+        for _ in 0..10 {
```

Comment:
should it quit if after 10 attempts there aren't any blocks, or it'll fail anyway?

Reply:
removed this waiting, user should just restart when bad things happen.